pkg/scaffold/cmd.go

```diff
@@ -45,7 +45,6 @@ import (
 
 	"{{ .Repo }}/pkg/apis"
 	"{{ .Repo }}/pkg/controller"
-
```

Comment:
Nit: Oh I prefer having a new line. Think it makes imports more clear, as these are the projects own imports and the rest are external, but it's just my personal preference. :)

Reply:
I do as well but unit tests are weirdly flaky for me with this newline. I can remove before merging.

Reply:
interesting :D wonder what is causing that?

Reply:
Also as a side note, nothing to do with this PR, but was thinking that files unit test is a bit long, think we could just compare the strings that are templated instead the entire file.

Reply:
> interesting :D wonder what is causing that?

I feel like I've seen similar issues with new lines in the scaffold imports and I had to make sure I was using the same version of go that we're using in CI (1.10.3)

Reply:
Yeah that's likely the issue. I'm going to remove this commit.